src/bhavana_assignment_14/ShopScanner.java

```diff
@@ -0,0 +1,92 @@
+package bhavana_assignment_14;
+
+import java.util.Scanner;
+
+import neha_Patil.shop;
+
+/*Create a system (Shop) which maintains each item's quantity.
+a. Initial stocks of Maggie: 50 Maggie packets 
+b. Initial stocks of Dosa : 43 packets 
+c. Initial stocks of pouches : 39 oil packets 
+d. Initial stocks of panipuri : 43 packets  
+e. Initial stocks of masala�s : 73 packets.
+
+Expectation:
+1) Only one Customer came for shopping, he picked different items with different quantities,
+ print "running out of stock" message if a particular item is sold out otherwise deduct requested 
+quantity from total quantity. Partial selling is allowed.  
+2) Create a method which shows only those item which is "Out of stock".
+3) Create a method which shows only those item which is "Available in stock" & respective quantities.
+*/
+public class ShopScanner {
+
+	int maggiQ=50, dosaQ=43,oilQ=39,panipuriQ=43,masalaQ=73;
+	
+	void getRequest() {
+		int maggi=0,dosa,oil,panipuri,masala;
+		Scanner sc=new Scanner(System.in);
```

Comment:
We should not ask, user to give inputs after calling methods, 
We should ask user to give input while calling methods, and we take input once only from main method not in every method